pages/common/tailscale-file.md

```diff
@@ -0,0 +1,13 @@
+# tailscale file
+
+> Send files across connected devices on a Tailscale network.
+> It currently does not support sending files to devices owned by other users even on the same Tailscale network.
+> More information: <https://tailscale.com/kb/1106/taildrop/>.
+
+- Send a file to a specific node:
+
+`sudo tailscale file cp {{path/to/file}} {{hostname_or_ip}}:`
```

Comment:
```suggestion
`sudo tailscale file cp {{path/to/file}} {{hostname|ip}}:`
```

I suggest always replace all `or` with `|` because it makes unambiguous multi word alternatives.